### ISSUE TITLE
Say all broken in latest snapshots.

### ISSUE THREAD

**Issue by tapper82** [nvaccess/nvda#14909]
### Steps to reproduce:
Open firefox use the nvda key insert and down arrow to say all.
### Actual behavior:

Speach stops after reading out the line you are focused on.
### Expected behavior:

NVDA should read until you press a key or reaching the bottom of the page.
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
It started in alpha-28166,e77bde9a and is still in alpha-28193,d1ce3bc9
#### Windows version:
Microsoft Windows
Version 22H2 (OS Build 22621.1635)

#### Name and version of other software in use when reproducing the issue:
Firefox 114.01 NVDA fails to say all anywhere not just in firefox
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
all snapshots since, nvda_snapshot_alpha-28161,b93fc546 
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by josephsl** [comment]
Hi, which speech synthesizer and/or add-ons do you have? CC @JCSTEh, looks like this could be something going on with a synth or an add-on after merging the API work. Thanks.

**Comment by tapper82** [comment]
Hi this is not just in firefox all so happens in windows to. EG when reading a txt file. First letter navigation is all so broken. I just spotted it. When on the desktop pressing F to jump to the firefox link does not say anything. All though it will still move focus to the firefox link.

**Comment by tapper82** [comment]
Installed Add-ons  list
Add-on Updater; Status: Enabled; Version: 23.05; Author: Joseph Lee
 Eloquence Synthesizer; Status: Enabled; Version: 0.220526.01; Author: NVDA User  
Tele NVDA remote assistance; Status: Disabled; Version: 2023.4.29; 
Windows App Essentials; Status: Enabled; Version: 23.05; Author: Joseph Lee 
Resource Monitor; Status: Enabled; Version: 23.05;

**Comment by josephsl** [comment]
Hi, okay, have you tried using different speech synthesizers and/or restarting after disabling add-ons to figure out what’s going on? Thanks.

**Comment by tapper82** [comment]
I am sorry for the noise. It's Eloquence Synthesizer that is messing things up. With all add-ons disabled things work as they should.

**Comment by Brian1Gaff** [comment]
Is this the old Sappi 4 via voice one? If it is it is now very old indeed 
and does tend to have, shall we say some'interesting' interactions.
 However if its the paid for one, folk tell me its fine.
 I don't use any Eloquence myself, as I find it very hard to understand, 
preferring Espeak for navigation and almost any good quality 'real' voice 
for  say all.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----
